_pages/en_US/seedminer-(mii).txt

```diff
@@ -0,0 +1,82 @@
+---
+title: "Seedminer (Mii)"
+---
+
+{% include toc title="Table of Contents" %}
+
+This method requires a reasonably powerful GPU (any dedicated graphics card made in the last few years) and the ability to use a command line. If you don't know what those are or do not have the hardware necessary, you can ask someone to do this for you at [Nintendo Homebrew on Discord](https://discord.gg/MWxPgEp). You will need to have the following information ready:
+  + Your Mii QR code, uploaded directly to Discord (do not take a screenshot or a photo of the image, just send the HNI_XXXX.JPG image itself)
+  + Your console's ID0 (the name of the 32-character folder directly inside of Nintendo 3DS - see https://3ds.hacks.guide/troubleshooting#seedminer if you have multiple)
+  + Your console model (e.g. 3DS, 3DS XL, 2DS, New3DS, New3DS XL, New2DS XL)
+  + Your console's approximate year of manufacturing (it may be present behind the frontplate or backplate; if you don't know, just guess - it's not super important)
+{: .notice--warning}
+
+{% capture technical_info %}
+<summary><em>Technical Details (optional)</em></summary>
+
+To install boot9strap on your device, we derive your device's unique encryption key. To accomplish this, we use a tool called Seedminer to calculate the data encryption key (movable.sed) for your device.
+
+Because we do not have access to the Friends List, we are deriving the first part of `movable.sed` using a Mii QR code instead of by performing a two-way friend exchange.
+
+For information on how Seedminer works, see [this presentation](https://zoogie.github.io/web/34⅕c3).
+
+{% endcapture %}
+<details>{{ technical_info | markdownify }}</details>
+{: .notice--info}
+
+## Instructions
+
+If you are on Linux, replace `py -3` with `python3` in all commands.
```

Comment:
This also applies to macOS. Not that I think macOS users will be able to follow this in a reasonable amount of time, but just for the sake of clarity.

macOS also comes with python3 preinstalled iirc.